src/set-pages-config.js

```diff
@@ -34,7 +34,8 @@ function getConfigParserSettings({ staticSiteGenerator, generatorConfigFile, sit
 
           // Disable server side image optimization too
           // https://nextjs.org/docs/api-reference/next/image#unoptimized
-          'experimental.images.unoptimized': true
+          'experimental.images.unoptimized': true,
+          'images.unoptimized': true
```

Comment:
Would be good to add a comment reminding us when this setting changed so we can deprecate the experimental one in the future. 👍🏻 

```suggestion
          // No longer experimental as of Next.js v12.3.0
          'images.unoptimized': true
```